src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner15.vue

```diff
@@ -0,0 +1,562 @@
+					<h2 class="strong" v-html="bannerHeadline"></h2>
+				</div>
+				<div class="swashie">
+					<kv-progress-circle class="swashie__progress-circle" :value="50" />
+					<swashie-face class="swashie__face" :percent-full="50" />
+				</div>
+				<!-- hiding everything with this template right now -->
+				<template v-if="false">
+					<kv-expandable
+						easing="ease-in-out"
+					>
+						<div class="sitewide-body small-12 columns"
+							v-show="open"
+						>
+							<div class="row">
+								<!-- eslint-disable-next-line max-len -->
+								<div class="hide-for-small show-for-medium medium-2 large-1 large-offset-1 columms thermometer-holder"
+									:title="`${ percentTowardGoal }% raised`"
+								>
+								</div>
+								<div class="small-12 medium-9 large-8 xlarge-7 columns">
+									<div>
+										<span
+											v-html="bannerBody"
+											class="message"
+										></span>
+									<!-- <span class="strong" v-html="bannerMatchingText"></span> -->
+									</div>
+									<div class="show-for-small hide-for-medium small-12 columns thermometer-holder">
+										<appeal-thermometer
+											:percent-toward-goal="percentTowardGoal"
+										/>
+									</div>
+									<div class="donation-buttons">
+										<ul>
+											<li v-for="(buttonAmount, index) in buttonAmounts"
+												:key="index"
+											>
+												<kv-button
+													class="mini custom-width"
+													@click.native.prevent.stop="updateDonationTo(buttonAmount)"
+													v-kv-track-event="[
+														'promo',
+														'click-amount-btn',
+														'AppealBanner',
+														buttonAmount
+													]"
+												>
+													${{ buttonAmount }}
+												</kv-button>
+											</li>
+										</ul>
+										<a
+											class="other-amount"
+											href="/donate/supportus"
+											@blur="validateInput"
+											v-kv-track-event="['promo', 'click-other', 'AppealBanner', 0]"
+										>Other amount
+										</a>
+									</div>
+								</div>
+							</div>
+						</div>
+					</kv-expandable>
+				</template>
+			</div>
+		</div>
+
+		<!-- GROW-230 -->
+		<div :class="this.open ? 'sitewide-overlay' : ''"
+			@click="toggleAccordion();"
+			v-kv-track-event="[
+				'Homepage',
+				'EXP-GROW-230-Sept2020',
+				'homepage-overlay-click-dismiss',
+			]"
+		>
+			<div class="overlay-content">
+				<h3>
+					Continue to site
+				</h3>
+			</div>
+		</div>
+		<kv-button
+			class="smaller dismiss-button"
+			:class="this.open ? 'white-button' : ''"
+			@click.native="donateButtonToggle();"
+		>
+			{{ this.open ? 'Donate later' : 'Donate' }}
+		</kv-button>
+	</div>
+</template>
+
+<script>
+import _forEach from 'lodash/forEach';
+import _get from 'lodash/get';
+import numeral from 'numeral';
+import store2 from 'store2';
+
+import appealBannerQuery from '@/graphql/query/appealBanner.graphql';
```

Comment:
The query in the global promo banner will get the setting and banner from contentful. We'll need to create a new query for this file that uses the new graphql query we've yet to build.